src/client/src/components/icons/LogoIcon.tsx

```diff
@@ -17,7 +17,7 @@ const LogoIcon: React.FC<Props> = ({ width, height, active }) => (
     <desc>Created with Sketch.</desc>
     <defs />
     <g
-      className={active ? 'svg-icon--active' : 'svg-icon'}
+      className={active ? "svg-icon--active" : "svg-icon"}
```

Comment:
suggest default width/height of 2 each for this icon (not sure why they didn't go with a default initially)
example usage in the app:
```
<LogoIcon width={1.2} height={1.2} active={active} />
```
from Launcher MainRoute